src/olympia/reviewers/tests/test_views.py

```diff
@@ -2616,7 +2616,7 @@ def test_item_history_pagination(self):
             str(author.get_role_display()),
             self.addon,
         )
-        with self.assertNumQueries(53):
+        with self.assertNumQueries(54):
```

Comment:
:grimacing:

Reply:
yeah... the incremental drip drip of extra queries.  We could move some of the information to be be async on page load (like the ADU numbers) - I'm not sure there's much scope for actually reducing the query count without dropping functionality unfortunately.